content/paths/sign_requests__post_sign_requests_id_resend.yml

```diff
@@ -0,0 +1,41 @@
+---
+operationId: post_sign_requests_id_resend
+
+summary: Resend Sign Request
+
+tags:
+  - Sign Requests
+
+x-box-tag: sign_requests
+
+description: |-
+  Resends a Sign Request to all signers that have not signed yet.
+
+parameters:
+  - $ref: '../attributes/sign_request_id.yml'
+
+responses:
+  200:
+    description: |-
+      Returns a Sign Request object.
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/SignRequest'
+  
+  404:
+    description: |-
+      Returns an error when the sign request could not be found or the
+      user does not have access to the sign request.
+    content:
+      application/json:
+        schema:
+          $ref: '#/components/schemas/ClientError'
+          
```

Comment:
I'm not sure if we ever really document this case, but since this will cause emails, I suppose, is there a rate limit for resending a single sign request?

Reply:
Yes we're working with security to get rate limiting setup for anything related to email notifications

Reply:
Great. Is the plan to alert the API caller? I could see it being done later in a queue, too.

Reply:
For the resend endpoint I just checked with the sign team and they have a restriction on their side to only allow the developer to resend 10 times in a minute - that check happens after the request returns though so the developer wouldn't be informed in that case.